### ISSUE TITLE
1.1.2 (x64) installer is broken

### ISSUE THREAD

**Issue by Balkoth** [microsoft/WindowsAppSDK#2713]
### Describe the bug

Deploying package: Microsoft.WindowsAppRuntime.1.1_1002.543.1943.0_x64__8wekyb3d8bbwe
Package deployment result : 0x80073cfb

https://task.ms/40349590

### Steps to reproduce the bug

Try installing 1.1.2 (x64)

### NuGet package version

1.1.2

### Windows version

Windows 10 version 21H2 (19044, November 2021 Update)

**Comment by Balkoth** [comment]
Happens when 1.1.1 was installed through the previous installer.

You can try to remove all the previous installed Windows App SDK components with
`get-appxpackage -AllUsers micro*win*appruntime* | remove-appxpackage -AllUsers`
but even this might fail because of dependencies between the packages.

In the end i had a client machine where i had to remove the packages manually one by one until this was cleaned up. I hope that in the future there is a simple way to uninstall a previous version, because the current process is horrendous.

**Comment by riverar** [comment]
@DrusTheAxe @gabbybilka Was the wrong runtime installer uploaded? Can confirm it's trying to install 1.1.1.

**Comment by BenJKuhn** [comment]
What this error seems to indicate is that there's a difference between the package provided by the installer and the package provided by the store, and you already have the updated store version installed. You can check this in powershell. Run 
```Get-AppxPackage Microsoft.WindowsAppRuntime.1.1 ```

You should see something like this:
```
Name              : Microsoft.WindowsAppRuntime.1.1
Publisher         : CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US
Architecture      : X64
ResourceId        :
Version           : 1002.543.1943.0
```

The version above is the correct version for 1.1.2 (The MSIX package is versioned with an encoded timestamp instead of patch number). If I then run the installer, I see the following error message:

> ErrorMessage: Deployment of package Microsoft.WindowsAppRuntime.1.1_1002.543.1943.0_x64__8wekyb3d8bbwe was blocked because the provided package has the same identity as an already-installed package but the contents are different. Increment the version number of the package to be installed, or remove the old package for every user on the system before installing this package.One or more install operations failed. Result: 0x80073cfb

The store unpacks & repacks contents of the MSIX as part of submission, and we've had some challenges in the past with slight differences between the version we put in the installer and the version that ultimately shows up in the store causing the install stack in Windows to believe they're not quite the same (but they should be). We're investigating whether this is the case.

Ben

**Comment by BenJKuhn** [comment]
I can confirm that my previous comment is accurate. The phone package ID changed (the store helpfully updates this) but we didn't update it from the 1.0 package version. So the store-serviced package doesn't match the WinAppSDK installer's version. The phone identity is an anachronism at this point, but the Windows install stack still recognizes it as a meaningful difference.

**Comment by riverar** [comment]
I hope we agree showing 1.1 to the dev/user is a bug.

**Comment by Balkoth** [comment]
A visual studio project updated to 1.1.2 does not run with the store updated version! It keeps insisting on that the required version is missing.

**Comment by BenJKuhn** [comment]
> I hope we agree showing 1.1 to the dev/user is also a bug.

@riverar, can you clarify what you mean here?

@Balkoth, We just approved the fix for this in shiproom today, so this will be in the next servicing update. I understand this is a significant nuisance. To work around it in the meantime, uninstall all App SDK 1.1 packages & dependencies, then run the 1.1.2 installer. Once you have the 1.1.2 version of the MSIX packages installed, the store updater won't swap it out for the store version, and the other deployment behaviors will work as expected (insatller, VS F5, etc.).

**Comment by riverar** [comment]
> @riverar, can you clarify what you mean here?

Users are trying to install the 1.1.2 Windows App Runtime. It shouldn't say or try to deploy `Microsoft.WindowsAppRuntime.1.1[...]`.

**Comment by gabbybilka** [comment]
This issue was fixed and released in Windows App SDK 1.1.3 - check out the [release notes](https://aka.ms/windowsappsdk/1.1/1.1.3/release-notes) and [NuGet](https://aka.ms/windowsappsdk/1.1/1.1.3) to try it out 🎉
